rbi/core/kernel.rbi

```diff
@@ -645,8 +645,8 @@ module Kernel
   end
   def to_enum(method=T.unsafe(nil), *args, &blk); end
 
-  sig {returns(String)}
-  def to_s(); end
+  sig {params(arg: Integer).returns(String)}
+  def to_s(*arg); end
```

Comment:
I think you want to be more precise here, since `x.to_s(5, 6)` would be a runtime error that would not be caught by Sorbet.  I know the docs say `*arg`, but I think that's an implementation detail of how the docs are generated.

This probably wants to be

```ruby
sig {params(base: Integer).returns(String)}
def to_s(base=10); end
```